api/SceHttpServer.js

```diff
@@ -88,10 +88,12 @@ class SceHttpServer {
         useCreateIndex: true
       })
       .then(() => {
-        console.log('MongoDB Connection Successful')
-        console.log()
+        // console.log('MongoDB Connection Successful')
```

Comment:
remove comments

Reply:
you need to keep the print statements here.
just change them to console.debug